src/Orchard.Web/Modules/Orchard.ImportExport/Services/SetupService.cs

```diff
@@ -101,6 +102,10 @@ public class SetupService : Component, ISetupService {
                     schemaBuilder.AlterTable("Orchard_Framework_DataMigrationRecord",
                         table => table.AddUniqueConstraint("UC_DMR_DataMigrationClass_Version", "DataMigrationClass", "Version"));
 
+                    // Create the distributed lock record schema.
```

Comment:
Yes absolutely, if I add back the `Migrations` class that will work fine as you say. If you don't care about the bit of code duplication (schema creation in `Migrations` as well as in `DistributedLockSchemaBuilder`) then I'd be happy to add it back and remove the call here. Just say the word.